test/naga/test_queue.cljc

```diff
@@ -7,65 +7,63 @@
 
 (t/use-fixtures :once st/validate-schemas)
 
-(deftest simple
-  "Test adding to an identity queue, without salience"
+(deftest adding-to-identity-queue-without-salience
   (let [data (shuffle (range 10))
-        queue (reduce q/add (q/new-queue) data)]
-    (is (= data (q/drain queue)))
-    (let [data2 (concat data (shuffle (range 10)))
-          queue (reduce q/add (q/new-queue) data2)]
-      (is (= data (q/drain queue))))))
+       queue (reduce q/add (q/new-queue) data)]
+   (is (= data (q/drain queue)))
+   (let [data2 (concat data (shuffle (range 10)))
+         queue (reduce q/add (q/new-queue) data2)]
+     (is (= data (q/drain queue))))))
 
-(deftest salience
-  "Test adding to an identity queue, with salience"
+(deftest adding-to-identity-queue-with-salience
   (let [data (shuffle (range 10))
-        queue (reduce q/add (q/new-queue identity identity) data)]
-    (is (= (range 10) (q/drain queue)))
-    (let [data2 (concat data (shuffle (range 10)))
-          queue (reduce q/add (q/new-queue identity identity) data2)]
-      (is (= (range 10) (q/drain queue))))))
+       queue (reduce q/add (q/new-queue identity identity) data)]
+   (is (= (range 10) (q/drain queue)))
+   (let [data2 (concat data (shuffle (range 10)))
+         queue2 (reduce q/add (q/new-queue identity identity) data2)]
+     (is (= (range 10) (q/drain queue2))))))
 
-(def test-data
+(def simple-test-data
   [{:id 1 :s 2}
    {:id 2 :s 2}
    {:id 3 :s 2}
    {:id 4 :s 1}
    {:id 5 :s 3}
    {:id 6 :s 2}])
 
-(deftest structure-elts
-  (let [queue (reduce q/add (q/new-queue :s :id) test-data)]
-    (is (= [4 1 2 3 6 5] (map :id (q/drain queue))))
-    (let [q2 (-> queue (q/add {:id 3 :s 2}) (q/add {:id 2 :s 3}))]
-      (is (= [4 1 2 3 6 5] (map :id (q/drain q2)))))))
+(deftest queue-with-simple-updates
+  (let [q1 (reduce q/add (q/new-queue :s :id) simple-test-data)
+        q2 (-> q1 (q/add {:id 3 :s 2}) (q/add {:id 2 :s 3}))]
+    (is (= [4 1 2 3 6 5] (map :id (q/drain q1))))
+    (is (= [4 1 2 3 6 5] (map :id (q/drain q2))))))
 
-(def update-data
+(def complex-update-data
```

Comment:
I would refer to this as `update-data` still, since it's the data used for updates. I don't believe it to have any complexity to it, since it contains just a numerical identifier, a salience value, and an atom that can record that an update occurred.